src/main/webapp/translate_sentiment.html

```diff
@@ -37,6 +37,9 @@ <h2>Translated Sentence Analyzer</h2>
         </form>
             <button onclick="getSentiment()">Submit</button>
             <div id="sentiment"></div>
+            <div id="progressBarGraph">
+                <div id="sentimentBarGraph" style="display:none;"></div>
+            </div>
```

Comment:
If you want, I can help center this and make it look a little prettier. Otherwise, LGTM

Reply:
If you can that would be awesome!! You can just add to this PR if you want or post the code in a comment.

Reply:
However, It does grow to eventually cover the whole div when the sentiment score is +1. It functions sort-of like a loading bar.

Reply:
### This is how it looks like with a score closer to +1.
![image](https://user-images.githubusercontent.com/16928852/114319272-669a5000-9ad6-11eb-8654-22fa9ba61d3e.png)

Reply:
Actually that works better! I'm fine with it!